### PR TITLE
Blog post: Return of experience on XMPP meetup in Krakow

### PR DESCRIPTION
**Pull request by Nyco**
Better late than never... ;-)

It is the result of a collaboration between Tigase and MongooseIM.

### CODE REVIEW

File: content/posts/blog/Return_of_experience_on_XMPP_meetup_in_Krakow.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+In May of 2017 a meetup was organized in Krakow, Poland. We announced it on the [XSF blog](https://xmpp.org/2017/04/first-krakow-xmpp-meetup/), and [on meetup.com](htpps://www.meetup.com/XMPP-Jabber-Messaging-Krakow/events/239508291/) (also a few social networks).  The reponse from the XMPP/Jabber community was very positive, as we were fresh off the heels of FOSDEM and many members had just recently met at the XMPP Summit.
+
+## Philosophy & goals
+
+Firstly, it was a cooperation between Tigase and MongooseIM. Both organizations provide open source XMPP server software and bring professional expertise on the protocol.  However, we wanted it to be a humble and fast organisation, rather than a meticulously planned event. Calling to mind the addage "done is better than perfect".  The ultimate goal of the meetup was to meet colleagues of the XMPP and Jabber community, brainstorm, and overall improve communication between users and developers of the protocol.  An additional benefit from the meeting so soon after the Summit was to strengthen the trend and popularity of XMPP meetups in the European Union.
```
Comment (SamWhited) [inline content/posts/blog/Return_of_experience_on_XMPP_meetup_in_Krakow.md]: > bring professional expertise on the protocol

Bring professional expertise to what?

> a humble and fast organisation

What does it mean for an organization to be humble or fast?

"organization"

> Calling to mind the addage "done is better than perfect".

This is not a complete sentence.

File: content/posts/blog/Return_of_experience_on_XMPP_meetup_in_Krakow.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Speakers and presentations
+
+Two speakers from Erlang Solutions and one from Tigase were on hand to provide presentations.
```
Comment (SamWhited) [inline content/posts/blog/Return_of_experience_on_XMPP_meetup_in_Krakow.md]: nit: "provide presentations" sounds weird to me, although I suppose it's not wrong. "make presentations"? Might just be me.

File: content/posts/blog/Return_of_experience_on_XMPP_meetup_in_Krakow.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Andrzej Wojcik of Tigase gave a presentation on "Why use XMPP for IoT?". A factual comparison: efficient and straight to the point as why other options do not provide a proper solution.  The alternatives, such as HTTP, AMQP, MQTT all have their advantages, but none bring a global solution as exhaustive as XMPP.  A demonstration was shown of a proof-of-concept of IoT devices being controlled VIA XMPP commands.
```
Comment (SamWhited) [inline content/posts/blog/Return_of_experience_on_XMPP_meetup_in_Krakow.md]: nit: "on 'Why use XMPP for IoT?'" sounds on; maybe replace "on" with "titled"? Again, might just be me.

> as why other

"as to why other" maybe? This whole stentence feels a bit oddly structured. Maybe: "It was a factual comparison, efficient and straight to the point, as to why other options do not provide a proper solution."

nit: "proper solution" also sounds vague, what *is* a "proper solution"? Probably fine, but maybe there's a better word. Complete solution?

> as exhaustive as XMPP.

"as exhaustive as XMPP's [solution]"

> VIA XMPP

"via XMPP"

File: content/posts/blog/Return_of_experience_on_XMPP_meetup_in_Krakow.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+![Szymon Mentel from Erlang Solutions](./Szymon_Mentel_MongooseIM.jpg)
+
+Piotr Nosek from Erlang Solutions, interacted with the audience around "Fantastic XMPP use-cases", showing the near-complete feature coverage of the open standards protocol and exploring what's missing or could be improved.  We hope that this discussion let to participant brainstorming and even more use cases for the protocol.
+
+![Piotr Nosek from Erlang Solutions](./Piotr_Nosek_MongooseIM.jpg)
+
+## Feedback and outcome
+
+Nice feedback from all attendees, all were happy to attend. There were some interesting question and answers following the presentation.  Followup discussions indicated that participants were engauged and were eager to contribute to the session.  It’s clear that these meetups can be far more engaging than a traditional chat session.  Being physically present seems to bring more wild and bright ideas to the front.
```
Comment (SamWhited) [inline content/posts/blog/Return_of_experience_on_XMPP_meetup_in_Krakow.md]: > Nice feedback from all attendees, all were happy to attend.

This is also a fragment; "we received nice feedback from all attendees; they were all happy to haev attended" maybe? Maybe not.

> engauged

engaged

File: content/posts/blog/Return_of_experience_on_XMPP_meetup_in_Krakow.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+![Andrzej Wojcik of Tigase](./Andrzej_Wojcik_Tigase.jpg)
+
+Szymon Mentel from Erlang Solutions, exposed the "ICE, STUN, and TURN" open standards implemented in MongooseICE, an open source server written in Elixir. The demo showed a movie and a camera stream from a Raspberry Pi. The stream was provided using Mangosta Android software, an open source XMPP client.
+
+![Szymon Mentel from Erlang Solutions](./Szymon_Mentel_MongooseIM.jpg)
+
+Piotr Nosek from Erlang Solutions, interacted with the audience around "Fantastic XMPP use-cases", showing the near-complete feature coverage of the open standards protocol and exploring what's missing or could be improved.  We hope that this discussion let to participant brainstorming and even more use cases for the protocol.
```
Comment (fenek) [inline content/posts/blog/Return_of_experience_on_XMPP_meetup_in_Krakow.md]: discussion ~~let~~ led?

**Comment by guusdk** [comment]
Hey @Nyco - sorry for me to take a long time to process this. There are two oddities with this post:

1. It doesn't show up in the blog. That's caused by the first few lines: they are not supposed to start with a + (plus) sign.
2. The images do not appear to show up, at least not when I'm running it locally. That could be a problem in my local instance, but I do note that yours are the only images in the `content/posts/blog` directory. I think they're supposed to go somewhere else (but I didn't spend the time to figure out where).

**Comment by Nyco** [comment]
Thx @guusdk ! ;-)
Plus signs fixed, image location is still to be worked on?

**Comment by guusdk** [comment]
Allrighty, it shows up on the Blog now (although you'll want to update the date to make it show up on top). The images don't show - I'd place them in a directory under https://github.com/xsf/xmpp.org/tree/master/content/images - that should work.

**Comment by Nyco** [comment]
@guusdk how do you move images with the GitHub web interface?

**Comment by guusdk** [comment]
I don't know. I'd check out the repository, apply the changes locally, and commit that.

**Comment by Nyco** [comment]
Please check again?

**Comment by guusdk** [comment]
Almost! This time I know how to fix it :)

The location of the images should be absolute, not relative. You can simply remove the first character of every path to an image. For example

```Andrzej Wojcik of Tigase```

should be

```Andrzej Wojcik of Tigase```

(You'll need to do that for every picture, obviously).

You might also want to update the blog date - but after that, we should finally be able to publish this. :)

**Comment by Nyco** [comment]
And now? ;-)

**Comment by Nyco** [comment]
- The pictures do not show up.
- And the second link in the intro points to an "htpps" URI rather than "https".
- "Follow us on https://www.meetup.com/XMPP-Jabber-Messaging-Krakow/" in not a link...

Should I re-open it?

**Comment by guusdk** [comment]
I don't think you can open a merged PR - but we should definitely fix it.

**Comment by Nyco** [comment]
How do we proceed?

**Comment by horazont** [comment]
Open a new PR fixing things \o/.

The second and third bullet points should be easy to fix, not sure about the images unfortunately. Maybe iteam can look into it? If the build works and shows the images locally, I suspect some frontend server configuration issue.

**Comment by guusdk** [comment]
All issues should now be fixed.

**Pull request merged**